mreg_cli/api/models.py

```diff
@@ -3263,7 +3263,7 @@ def get_list_by_any_means(
         # If we found a CNAME, get the host it points to. We're not interested in the
         # CNAME itself.
         if cname is not None:
-            host = cls.get_list_by_id(cname.host)
+            hosts = cls.get_list_by_id(cname.host)
```

Comment:
Oddly, this did not change the test results. Perhaps missing coverage in tests?

Reply:
> Perhaps missing coverage in tests?

😱 Can't be. Ever.